Lib/asyncio/tasks.py

```diff
@@ -33,6 +33,16 @@ def current_task(loop=None):
 
 def all_tasks(loop=None):
     """Return a set of all tasks for the loop."""
+    if loop is None:
+        loop = events.get_event_loop()
```

Comment:
`get_running_loop()` maybe?
If `all_tasks()` is not the same as `Task.all_tasks()` anymore...

Reply:
I'd go for running, yes; it was my original request.  Let me update the PR.